### PR TITLE
ref: Skip flaky backfill test

### PR DESCRIPTION
**Pull request by jangjodi**
Skip flaky backfill test until I can properly replicate it and fix it

### CODE REVIEW

File: tests/sentry/tasks/test_backfill_seer_grouping_records.py [hunk 1]
```diff
@@ -851,6 +852,9 @@ def test_backfill_seer_grouping_records_groups_has_invalid_neighbor(
         last_processed_index = int(redis_client.get(make_backfill_redis_key(self.project.id)) or 0)
         assert last_processed_index == len(groups)
 
+    @unittest.skip(
```
Comment (asottile-sentry) [inline tests/sentry/tasks/test_backfill_seer_grouping_records.py:855]: this should not use unittest.skip, we're using `pytest`

File: tests/sentry/tasks/test_backfill_seer_grouping_records.py [hunk 2]
```diff
@@ -874,10 +878,7 @@ def test_backfill_seer_grouping_records_multiple_batches(self, mock_post_bulk_gr
 
         mock_post_bulk_grouping_records.return_value = {"success": True, "groups_with_neighbor": {}}
 
-        with TaskRunner(), patch(
-            "sentry.tasks.backfill_seer_grouping_records.backfill_seer_grouping_records.apply_async",
-            wraps=backfill_seer_grouping_records(self.project.id, None),
-        ):
+        with TaskRunner():
```
Comment (asottile-sentry) [inline tests/sentry/tasks/test_backfill_seer_grouping_records.py:881]: this seems unrelated to skipping the test

**Review by asottile-sentry — APPROVED** [review]

**Review by JoshFerge — APPROVED** [review]
